### ISSUE TITLE
Set index error: Data type LowCardinality(Nullable(String)) must be deserialized with multiple streams

### ISSUE THREAD

**Issue by reflection** [ClickHouse/ClickHouse#4486]
Creating experimental set index on LowCardinality(Nullable(String)) column causes exception when filtering with where clause:

```
SET allow_experimental_data_skipping_indices=1;

CREATE TABLE `test_noidx` (
  timestamp         DateTime,
  action            LowCardinality(Nullable(String)),
  user              LowCardinality(Nullable(String))
) ENGINE=MergeTree
  PARTITION BY toYYYYMMDD(timestamp)
  ORDER BY (timestamp, action, cityHash64(user))
  SAMPLE BY cityHash64(user);
INSERT INTO test_noidx VALUES (1550883010, 'subscribe', 'alice');
INSERT INTO test_noidx VALUES (1550883020, 'follow', 'bob');

SELECT * FROM test_noidx WHERE user = 'alice';
┌───────────timestamp─┬─action────┬─user──┐
│ 2019-02-23 00:50:10 │ subscribe │ alice │
└─────────────────────┴───────────┴───────┘

1 rows in set. Elapsed: 0.079 sec.

CREATE TABLE `test` (
  timestamp         DateTime,
  action            LowCardinality(Nullable(String)),
  user              LowCardinality(Nullable(String)),
  INDEX test_user_idx (user) TYPE set(0) GRANULARITY 8192
) ENGINE=MergeTree
  PARTITION BY toYYYYMMDD(timestamp)
  ORDER BY (timestamp, action, cityHash64(user))
  SAMPLE BY cityHash64(user);
INSERT INTO test VALUES (1550883010, 'subscribe', 'alice');
INSERT INTO test VALUES (1550883020, 'follow', 'bob');

SELECT * FROM test WHERE user = 'alice';

Received exception from server (version 19.3.5):
Code: 385. DB::Exception: Received from localhost:9000, 127.0.0.1. DB::Exception: Data type LowCardinality(Nullable(String)) must be deserialized with multiple streams.

0 rows in set. Elapsed: 0.016 sec.
```

**Comment by reflection** [comment]
Any update on this?  We'd love to test out the new data skipping indices feature, but this is a blocker since all of our tables utilize the LowCardinality(Nullable(String)) type extensively.  Thanks!

**Comment by alexey-milovidov** [comment]
It's trivial issue, easy to fix.

**Comment by KochetovNicolai** [comment]
I'll fix it tomorrow.

**Comment by orloffv** [comment]
@KochetovNicolai fixed yet?

**Comment by KochetovNicolai** [comment]
Yes, fixed by #4486
